daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/value/Value.scala

```diff
@@ -4,13 +4,13 @@
 package com.digitalasset.daml.lf.value
 
 import com.digitalasset.daml.lf.archive.LanguageVersion
-import com.digitalasset.daml.lf.data.Ref.Identifier
+import com.digitalasset.daml.lf.data.Ref.{Identifier, Name}
 import com.digitalasset.daml.lf.data._
+import com.digitalasset.daml.lf.data.Ref.Name.equalInstance
```

Comment:
```suggestion
```

to match the previous suggestion about instance definition.

Reply:
If I remove this import it does not compile.